larger_than_memory.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<p>Might show more functions, depending on which ones are available or can be made available for dask with minor adjustments.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>This might be good enough. I don't have the impression though that this tutorial gets the point across that this scales with dataset size. Like we never show RAM usage or show that this works in 100 million patients (don't do that for a tutorial). IDK whether the reviewer would read this and think: "Yes, I get it."</p>